src/Composer/Command/RemoveCommand.php

```diff
@@ -117,24 +125,15 @@ protected function interact(InputInterface $input, OutputInterface $output)
             foreach ($lockedPackages as $package) {
                 $unused[] = $package->getName();
             }
-            $input->setArgument('packages', array_merge($input->getArgument('packages'), $unused));
+            $packages = array_merge($packages, $unused);
 
-            if (count($input->getArgument('packages')) === 0) {
+            if (count($packages) === 0) {
                 $this->getIO()->writeError('<info>No unused packages to remove</info>');
                 $this->setCode(static function (): int {
                     return 0;
                 });
```

Comment:
This can now be simplified:

```suggestion
                $this->getIO()->writeError('<info>No unused packages to remove</info>');

                return 0;
```